src/Traits/FixturesTrait.php

```diff
@@ -188,8 +190,8 @@ protected function getTables(): array
 
     protected function exportContent($content, string $fixture): void
     {
-        if (env('FAIL_EXPORT_JSON', true)) {
-            $this->fail(preg_replace('/[ ]+/mu', ' ',
+        if (config('FAIL_EXPORT_JSON', true)) {
+            throw new UnexpectedExportException(preg_replace('/[ ]+/mu', ' ',
                 'Looks like you forget to remove exportJson. If it is your local environment add 
```

Comment:
let's move message to the exception class

Reply:
@DenTray Done